app/controllers/batch_update_tools_controller.rb

```diff
@@ -1,3 +1,4 @@
+require 'json'
```

Comment:
I read that you didn't need to `require` code that was under the app folder, but I thought you still needed it for other libraries?

Reply:
In this case `json` is a 3rd party library that is pulled in via the `Gemfile` (which is explicitly loaded in [`/config/application.rb`](https://github.com/nova-labs/watto/blob/main/config/application.rb#L7). If you have code in the project (say in `/lib`) then it would need to be explicitly required.